tests/test_evaluate.py

```diff
@@ -150,7 +151,7 @@ def test_absolute_average_fold_error():
 
 
 def test_metric_y_types(
-    tmpdir: str, test_single_task_benchmark_clf: SingleTaskBenchmarkSpecification, test_data: Dataset
+    tmpdir: str, test_single_task_benchmark_clf: SingleTaskBenchmarkSpecification, test_data: DatasetV1
```

Comment:
I know we can use the `tmpdir` argument with Pytest for getting easy access to clean directories, but is it needed in the few functions in this file that don't access it?

Reply:
No, it's not! 👍